src/loarconf_sample_h

```diff
@@ -1,89 +1,88 @@
-#ifndef __LORACONF_H__

```

Comment:
Why did you change this filename? [suggestion: keep it]

Reply:
checked, file is correct I didn't change it...

    #ifndef __LORACONF_H__
    #define __LORACONF_H__

Reply:
ok now renamed to loraconf_sample.h